README.md

```diff
@@ -179,6 +182,10 @@ This is not an officially supported Google product.
 
 ## Release Notes
 
+### v0.0.5
+
+- Change type of integrityBlockSign.key to be KeyObject instead of string.
```

Comment:
I think calling out that this is a breaking change would be good - i.e., by writing `BREAKING CHANGE: ...` and increasing the version number to `0.1.0` instead of `0.0.5`.

Reply:
Done!